tests/tools/cohort_tracking/test_cohort_tracking.py

```diff
@@ -0,0 +1,145 @@
+from pathlib import Path
+
+import pytest
+
+import ehrapy as ep
+from ehrapy.io._read import read_csv
+
+CURRENT_DIR = Path(__file__).parent
+_TEST_DATA_PATH = f"{CURRENT_DIR.parent}/test_data_features_ranking"
+_TEST_IMAGE_PATH = f"{CURRENT_DIR.parent}/_images"
+
+
+@pytest.fixture
+def adata_mini():
+    return read_csv(f"{_TEST_DATA_PATH}/dataset1.csv", columns_obs_only=["glucose", "weight", "disease", "station"])
+
+
+@pytest.mark.parametrize("columns", [None, ["glucose", "weight", "disease", "station"]])
+def test_CohortTracker_init_vanilla(columns, adata_mini):
+    ct = ep.tl.CohortTracker(adata_mini, columns)
+    assert ct._tracked_steps == 0
+    assert ct.tracked_steps == 0
+    assert ct._tracked_text == []
+    assert ct._tracked_operations == []
+
+
+def test_CohortTracker_type_detection(adata_mini):
+    ct = ep.tl.CohortTracker(adata_mini, ["glucose", "weight", "disease", "station"])
+    assert set(ct.categorical) == {"disease", "station"}
+
+
+def test_CohortTracker_init_set_columns(adata_mini):
+    # limit columns
```

Comment:
```suggestion
    # limit columns
```

Don't say what. Only say why or remove the comment